src/main/scala/ai/datahunters/md/workflow/MainExtractionWorkflow.scala

```diff
@@ -29,21 +30,23 @@ class MainExtractionWorkflow(config: ProcessingConfig,
 
 
   override def run(): Unit = {
-
+    val mandatoryTagConfig = MandatoryTagsConfig.build(config)
+    var mandatoryTagsFilter: Option[NotEmptyTagFilter] = None
```

Comment:
@dawidsau You can use shorter version instead of whole if below:
```
val mandatoryTagsFilter = mandatoryTagConfig.dirTags.map(d => new NotEmptyTagFilter(d))
```